package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "observable-input",
-  "version": "1.0.0",
-  "description": "angular input properties as observable streams",
+  "name": "ng-observable-input",
+  "version": "1.0.2",
```

Comment:
Mostly when you open PRs the repo owner will be responsible for incrementing the package version, so you'd want to avoid having this in your PR.